kaldb/src/test/java/com/slack/kaldb/writer/LogMessageWriterImplTest.java

```diff
@@ -163,13 +163,6 @@ public void testApiLogMessageInsertion() throws Exception {
     assertThat(searchChunkManager(testIndex, "").hits.size()).isEqualTo(1);
     assertThat(searchChunkManager(testIndex, "http_method:POST").hits.size()).isEqualTo(1);
     assertThat(searchChunkManager(testIndex, "http_method:GET").hits.size()).isEqualTo(0);
-    assertThat(searchChunkManager(testIndex, "method:callbacks*").hits.size()).isEqualTo(1);
```

Comment:
we're not really testing search behaviour here. None of these will ever fail so removing some . Just makes it tough to modify any data

Reply:
Can we please ensure that we are running similar searches elsewhere in the codebase? Otherwise we may be opening the door for bugs.